### PR TITLE
Update README and add a LICENSE

### PR DESCRIPTION
**Pull request by ethanliu24**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,149 @@
...
+ConnectHub is a discussion forum where users can engage in meaningful conversations. You can sign up and create your account to become part of the community. Once you're in, you can easily create, edit, and delete your own posts. To make your experience even better, ConnectHub lets you customize your feed with category filters, so you can focus on posts that match your interests.​
+
+This project is a hybrid of a Swing app and a React app with SpringBoot as the backend. You can choose to run either versions of the app.
+
+
+## Features
+- [x] Create posts online
+- [x] Manage your posts
+- [x] View posts created by others
+- [x] Share your or others' posts
```
Comment (Karththigan) [inline README.md:36]: do we want to also add filtering

File: README.md [hunk 2]
```diff
@@ -1,13 +1,149 @@
-# connecthub
+# ConnectHub
 
-# Run the web version locally
+***
 
-This project is unlikely to be deployed, everything will happen locally. To run the website:
+## Overview
+ConnectHub is a forum for meaningful discussions, offering easy post management and personalized feeds based on your interests.
```
Comment (Karththigan) [inline README.md:6]: think we should add 
  Says why the project was made (CSC207project)
 Gives the user a general sense of what problem this project solves and whether it is a useful project for them
(this project helps connect students/etc. together and was helpful for learning java LOL or something 💀 )

File: README.md [hunk 3]
```diff
@@ -1,13 +1,149 @@
...
 
-1. `npm install` to install all front end dependencies (make sure you have `npm` installed, this project uses the newest node version `v20.17.0`)
-2. Load the `pom.xml` to install all back end dependencies
-3. `npm run watch` to build all static assets
-4. Run the `WebServer.java` script in `src/main/java/app`
-4. navigate to `http://localhost:8080/signup` to start (sign up first!)
+## Table of Contents
+1. [Description](#description)
```
Comment (Karththigan) [inline README.md:11]: according to the checklist we need links to all the sections in the readme : )

File: README.md [hunk 4]
```diff
@@ -1,13 +1,149 @@
...
+
+
+## Usage
+After you successfully ran the application (either swing or web), you can either sign up or log in. Once in the home page, you can see a list of posts that are posted by others. Click on them to see the posts individually.
+
+You can also find a button to create post. The new post created will be updated in the home page. Additionally on the home page, you can filter posts by clicking on the topics on the left.
+
+You can also delete the posts you created by going to your post and clicking the delete button.
+
+Note that post filtering and deletion features is not available on the web version.
```
Comment (Karththigan) [inline README.md]: think we should move this note to the "Features" section

**Review by Karththigan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ethanliu24) [reply README.md:11]: oops forgot about this :)

File: README.md (hunk 2 above)
Comment (ethanliu24) [reply README.md:6]: yea whatever this is so corny lmao

**Review by Karththigan — APPROVED** [review]
LGTM!

**Pull request merged**